### PR TITLE
Footer Accessibility Fix

### PR DESCRIPTION
**Pull request by MaidenManitoba**
Updated accessibility features.
Changed header.
Added aria-label to footer input.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,190 +1,202 @@
...
-					<button class="btn-la">Land Acknowledgement</button>
+					<!-- form -->
+					<form action="" class="">
+						<h2 class="h4-styles">Stay up to date with us</h2>
+						<p>
+							Register with our newsletter to receive updates on our website,
+							events, and more!
+						</p>
+						<div class="text-input">
+							<label class="hidden" for="newsletter-input">Newsletter</label>
```
Comment (SolivanLau) [inline index.html:82]: Thanks for adding in `<label>` elements! All `<input>` elements inside of a `<form>`  element need a label to describe its purpose!

File: index.html [hunk 2]
```diff
@@ -1,190 +1,202 @@
...
+							<input
+								type="email"
+								name="newsletter-input"
+								id="newsletter-input"
+								size="40"
+								placeholder="Enter your email"
+								aria-label="newsletter"
```
Comment (SolivanLau) [inline index.html:89]: nice on the aria label! more about accessibly creating inputs here: https://www.w3.org/WAI/tutorials/forms/labels/

**Review by SolivanLau — APPROVED** [review]
looks great! Left a few comments reviewing the discussion on creating accessible forms in html w `<label>` elements

**Review by maxharrisnet — APPROVED** [review]
Great work!

**Pull request merged**